loky/backend/context.py

```diff
@@ -67,6 +69,43 @@ def get_context(method="loky"):
                              "methods are {}".format(method, METHODS))
 
 
+def cpu_count():
+    """Return the number of CPUs the current process can use.
+    """
+    import math
+
+    try:
+        cpu_count_mp = mp.cpu_count()
```

Comment:
It can return `None`. This should be handled too.

Reply:
`os.cpu_count` can return `None`, but in this case `multiprocessing.cpu_count` [raises an exception](https://github.com/python/cpython/blob/3972628de3d569c88451a2a176a1c94d8822b8a6/Lib/multiprocessing/context.py#L43) that is handled ...